entwatch/ze_grau_a03_4f.cfg

```diff
@@ -0,0 +1,76 @@
+//Even though all items are set to be hidden on this entwatch, it is so that ebanned players cant pick up powers as z or ct
+
+"entities" {
+	"0" {
+		"name"              "Heal"
+		"shortname"         "Heal"
+		"color"             ""
+		"buttonclass"       ""
+		"filtername"        ""
+		"hasfiltername"     "false"
+		"blockpickup"       "false"
+		"allowtransfer"     "true"
+		"forcedrop"         "true"
+		"chat"              "false"
+		"hud"               "false"
+		"hammerid"          "628937"
+		"mode"              "0"
+		"maxuses"           "0"
+		"cooldown"          "0"
+		"maxamount"         "1"
+	}
+	"1" {
+		"name"              "Zombie Powerup 3"
+		"shortname"         "ZM 3"
+		"color"             ""
+		"buttonclass"       ""
+		"filtername"        "item3_gr"
+		"hasfiltername"     "true"
+		"blockpickup"       "false"
+		"allowtransfer"     "false"
+		"forcedrop"         "false"
+		"chat"              "true"
+		"hud"               "true"
+		"hammerid"          "377002"
+		"mode"              "0"
+		"maxuses"           "0"
+		"cooldown"          "0"
+		"maxamount"         "1"
+	}
+	"2" {
+		"name"              "Zombie Powerup 1"
+		"shortname"         "ZM 1"
+		"color"             ""
+		"buttonclass"       ""
+		"filtername"        "item1_gr"
+		"hasfiltername"     "true"
+		"blockpickup"       "false"
+		"allowtransfer"     "true"
+		"forcedrop"         "true"
+		"chat"              "true"
+		"hud"               "true"
+		"hammerid"          "298159"
+		"mode"              "0"
+		"maxuses"           "0"
+		"cooldown"          "0"
+		"maxamount"         "1"
+	}
+	"3" {
+		"name"              "Zombie Powerup 2"
+		"shortname"         "ZM 2"
+		"color"             ""
+		"buttonclass"       ""
+		"filtername"        "item2_gr"
+		"hasfiltername"     "true"
+		"blockpickup"       "false"
+		"allowtransfer"     "true"
+		"forcedrop"         "true"
+		"chat"              "true"
+		"hud"               "true"
+		"hammerid"          "298201"
+		"mode"              "0"
+		"maxuses"           "0"
+		"cooldown"          "0"
+		"maxamount"         "1"
+	}
+}
```

Comment:
This one can be kept, but I don't understand the comment. How does hiding the item relate to preventing ebanned players from grabbing the item? That should work regardless.

Reply:
I have heal hidden on this since I viewed it as rather useless to display on the scoreboard since it is a passive items (and as far as I knew, ebanned players are only stopped from picking up weapons whose hammer ids are listed in the entwatch config, but I may be wrong about that). As for the comment saying all items are hidden, that is just because I used to have all zombie items not show too, but edited that before adding to this repo and forgot to change the commit. I'll change them to all show and remove the comment.